tool/teleport-update/main.go

```diff
@@ -141,14 +143,16 @@ func Run(args []string) int {
 	updateCmd := app.Command("update", "Update the agent to the latest version, if a new version is available.")
 	updateCmd.Flag("now", "Force immediate update even if update window is not active.").
 		Short('n').BoolVar(&ccfg.UpdateNow)
-	updateCmd.Flag("self-setup", "Use the current teleport-update binary to create systemd service config for auto-updates.").
+	updateCmd.Flag("self-setup", "Use the current teleport-update binary to create systemd service config for auto-updates and verify the Teleport installation.").
 		Short('s').Hidden().BoolVar(&ccfg.SelfSetup)
 
 	linkCmd := app.Command("link-package", "Link the system installation of Teleport from the Teleport package, if auto-updates is disabled.")
 	unlinkCmd := app.Command("unlink-package", "Unlink the system installation of Teleport from the Teleport package.")
 
-	setupCmd := app.Command("setup", "Write configuration files that run the update subcommand on a timer.").
+	setupCmd := app.Command("setup", "Write configuration files that run the update subcommand on a timer and verify the Teleport installation.").
 		Hidden()
+	setupCmd.Flag("restart", "Restart the Teleport agent.").
+		Short('r').BoolVar(&ccfg.Restart)
```

Comment:
Can we name the flag reload and also explain what happens when the flag is not set. The way i understood it initially was: `--restart` -> restart, else reload. While in reality this is: `--restart` -> reload, else we just setup.